ui/src/Connection.elm

```diff
@@ -115,11 +110,19 @@ update msg model token =
             ( { model | channels = updatedChannel }, Cmd.map ChannelMsg channelCmd )
 
         CheckThing id ->
-            if List.member id model.checkedThingsIds then
-                ( { model | checkedThingsIds = List.Extra.remove id model.checkedThingsIds }, Cmd.none )
+            ( { model | checkedThingsIds = checkEntity id model.checkedThingsIds }, Cmd.none )
 
-            else
-                ( { model | checkedThingsIds = id :: model.checkedThingsIds }, Cmd.none )
+        CheckChannel id ->
+            ( { model | checkedChannelsIds = checkEntity id model.checkedChannelsIds }, Cmd.none )
+
+
+checkEntity : String -> List String -> List String
+checkEntity id checkedEntitiesIds =
+    if List.member id checkedEntitiesIds then
+        List.Extra.remove id checkedEntitiesIds
+
+    else
+        id :: checkedEntitiesIds
 
 
 view : Model -> String -> Html Msg
```

Comment:
Why do you need token as argument in view? It's not used

Reply:
Removed in the last commit.